pyews/ews/autodiscover.py

```diff
@@ -1,68 +1,74 @@
 # -*- coding: utf-8 -*-
```

Comment:
Hey, sorry I cannot accept these changes. The result is not pretty. Please do not make gratuitous formatting changes to the code.